### ISSUE TITLE
Continuous tracking sometimes not complete 

### ISSUE THREAD

**Issue by Egregius** [owntracks/ios#819]
Hi all,

Yesterday I noticed that Owntracks stopped recording at sometime and them automatically restarted a while later. The first lines of this logfiles is where we were walking. During the walk the recording stopped with just a [Connection] idle message. 
Then about 90 minutes later it restarted automatically while driving. It could be that that was the moment I started Guru Maps, Google Maps was already running from the start of the drive for navigation.

I have no automations, just a "Home|1|2" location to start the tracking automatically whenever I leave the home. 

I have now set the locatorInterval down to 60, which was on 600. Will that be an improvement for more stability?

I'm using Owntracks 18.0.4 on iOS 18.1 (22B82).
I don't think the iOS beta has much to do with it, I've seen this behavior several times in the past.

[Owntracks.log](https://github.com/user-attachments/files/17537753/Owntracks.log)

Thanks for your reply!

**Comment by ckrey** [comment]
@Egregius please check your setting for "Downgrade". The app will downgrade from "Move" mode to "Significant Mode" when the battery level drops below the downgrade percentage. The problem might be related to this.

**Comment by Egregius** [comment]
Downgrade is set to 40. My battery never went below 60.
![photo_2024-10-28 10 20 15](https://github.com/user-attachments/assets/f7c98d97-0a10-4118-aa95-90dc88e83d71)

**Comment by ckrey** [comment]
Is this still happening with an non-betaiOS 18.x.x?

**Comment by Egregius** [comment]
Nope, not seen this issue in a long time. I even forgot about it :)
